.github/workflows/nia-e2e.yaml

```diff
@@ -21,44 +21,134 @@ jobs:
     - name: Setup Anbox Cloud
       uses: canonical/anbox-cloud-github-action@dc99b94280cfcf44a86b62aeac25ba34f14725e0
       with:
-        channel: latest/stable
+        channel: 1.28/edge/amc-connect
 
     - name: Tune Anbox Cloud
       run: |
         amc config set container.security_updates false
 
+    - name: Setup remote amc
+      env:
+        REMOTE_AMC_URL: ${{ vars.REMOTE_AMC_URL }}
+        REMOTE_SERVER_CERT: ${{ secrets.REMOTE_SERVER_CERT }} # ams.crt
+        REMOTE_CLIENT_CERT: ${{ secrets.REMOTE_CLIENT_CERT }} # client.crt
+        GH_RUNNER_CERT: ${{ secrets.GH_RUNNER_CERT }} # runner.crt
+        GH_RUNNER_KEY: ${{ secrets.GH_RUNNER_KEY }} # runner.key
+        
+      run: |
+        set -x
+
+        url=${REMOTE_AMC_URL}
+        if [ -z "${url}" ]; then
+          url="https://$(amc config show | grep -Po 'core\.https_address:\s*\K.*')"
+        fi
+
+        if [ -n "${REMOTE_SERVER_CERT}" ]; then 
+          mkdir -p "$HOME/snap/anbox-cloud-appliance/current/client/servercerts"
+          echo "${REMOTE_SERVER_CERT}" > "$HOME/snap/anbox-cloud-appliance/current/client/servercerts/server.crt"
+          amc config trust add "$HOME/snap/anbox-cloud-appliance/current/client/servercerts/server.crt"
```

Comment:
This is not necessary. The certificate of the remote server is only needed on the client, not the server.